include/ensmallen_bits/callbacks/early_stop_at_min_loss.hpp

```diff
@@ -68,6 +101,9 @@ class EarlyStopAtMinLoss
   }
 
  private:
+  //! False if the first constructor is called, true is the user passed a lambda. 
+  bool callback_used;
```

Comment:
```suggestion
  bool callbackUsed;
```
We should avoid underscores in the variable names (as per the mlpack style guide). :+1: